dev-guide/deploy.md

```diff
@@ -1,18 +0,0 @@
----
-title: Build for Deployment
```

Comment:
All documents in the `dev-guide` directory are not parsed into web pages in docs.pingcap.com.